seed/main/hi_Latn.xml

```diff
@@ -5279,11 +5278,6 @@ annotations.
 				<unitPattern count="other">{0} gal</unitPattern>
 				<perUnitPattern>{0}/gal</perUnitPattern>
 			</unit>
-			<unit type="volume-quart">
-				<displayName>qts</displayName>
-				<unitPattern count="one">{0} qt</unitPattern>
-				<unitPattern count="other">{0} qt</unitPattern>
-			</unit>
```

Comment:
this conflicts with en_001 (and en_IN) where these are *US* quarts and imperial are just *quarts*. So, deleted.